src/com/esotericsoftware/yamlbeans/SafeYamlConfig.java

```diff
@@ -0,0 +1,44 @@
+package com.esotericsoftware.yamlbeans;
+
+
+/**
+ * SafeYamlConfig extends YamlConfig and hard codes the read anchor and read class tag flags to false.
+ * When these flags are enabled, it is possible to perform a deserialization attack if the Yaml being parsed is from an
+ * untrusted source.
+ * Using SafeYamlConfig is the equivalent of using YamlConfig after setting
+ *    yamlConfig.readConfig.setAnchors(false);
+ *    yamlConfig.readConfig.setClassTags(false);
+ *
+ * It should be noted by setting these two values neither anchors or specifying class names are supported.
+ * It is still possible to deserialize back to a specific object, but you need to specify the Class type in the code.
+ * e.g
+ *   SafeYamlConfig yamlConfig = new SafeYamlConfig();
+ *   YamlReader reader = new YamlReader(yamlData.toString(),yamlConfig);
+ *   Data data = reader.read(Data.class);
+ *
+ */
+public class SafeYamlConfig extends YamlConfig {
+
+
+    public SafeYamlConfig () {
+        super();
+        super.readConfig = new SafeReadConfig();
+    }
+
+    static public class SafeReadConfig extends ReadConfig {
+
+        public SafeReadConfig(){
+            super.anchors = false;
+            super.classTags = false;
+        }
+
+        @Override
+        public void setClassTags(boolean anchors) {
```

Comment:
It would be better to throw unsupported operation than to silently ignore a method call that is invalid. The rest looks OK. I'll run the source formatter after merge.

Reply:
Yes I think you are right.  It would have to be an unchecked exception. Will add it.